### ISSUE TITLE
[CLOSED] estimate cost summary

### ISSUE THREAD

**Issue by adambreznicky** [TNRIS/dataHub#37]
<a href="https://github.com/adambreznicky"><img src="https://avatars3.githubusercontent.com/u/5650145?v=4" align="left" width="96" height="96" hspace="10"></img></a> **Issue by [adambreznicky](https://github.com/adambreznicky)**
_Tuesday Jan 15, 2019 at 16:53 GMT_
_Originally opened as https://github.com/TNRIS/data.tnris.org/issues/37_

----

textual blurb at bottom of populated shopping cart order form needs to be finalized. this text is supposed to be supplied by @ThorTheDestroyer as per a previous meeting. if we don't have permanent charges/cost for services or there is no additional information we want to relay to customers prior to their order being placed, then this chunk can go away

**Comment by adambreznicky** [comment]
<a href="https://github.com/ThorTheDestroyer"><img src="https://avatars1.githubusercontent.com/u/8398036?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [ThorTheDestroyer](https://github.com/ThorTheDestroyer)**
_Tuesday Jan 15, 2019 at 19:39 GMT_

----

Are you looking for pricing descriptions/ Listed product cost or equations for calculating data costs? Or Both?

**Comment by adambreznicky** [comment]
<a href="https://github.com/adambreznicky"><img src="https://avatars3.githubusercontent.com/u/5650145?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [adambreznicky](https://github.com/adambreznicky)**
_Tuesday Jan 15, 2019 at 19:59 GMT_

----

the more information we can provide users up front the better. it means less questions coming in as well as erroneous orders where people don't realize what exactly they're asking for. i'm asking for any and all information you want to provide to a user right before they submit an order.

one thing we had discussed in a previous meeting is relaying cost information to the customers. no, i don't think we need an equation but more of a breakdown of what they will be charged including cost of the 1tb and 64gb hard drives, staff labor charges, machine "computer" charges, data size charges, etc. We could also display information related to typical turn around time and specifically when tnris will contact them with the final cost. 

if all of the charges/pricing sheet are listed, one day we could conceptually do the math and estimate the total cost to display. even in the mean time we could tell them the for-sure cost of their choices in hard drive, delivery, and full coverage datasets. 

tldr; i'm asking for any and all information you want to provide to a user right before they submit an order.

**Comment by adambreznicky** [comment]
<a href="https://github.com/adambreznicky"><img src="https://avatars3.githubusercontent.com/u/5650145?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [adambreznicky](https://github.com/adambreznicky)**
_Tuesday Jan 15, 2019 at 20:00 GMT_

----

![Screenshot from 2019-01-15 13-59-44.png](https://waffleio-direct-uploads-production.s3.amazonaws.com/uploads/5972568472d14a001f805e0f/125516c66e82c728ace21e0d46db978826878dba87e6ab03a24fff99691477227049f37a3dd4b668ee11365456424aa853444d04bea6d37ef5f22e7a985958ecd06a5627c16940abe98a16bf00b2768e4e87beb9d85edcfa7f74b9dfd0126754505a9bf54f37a1934acd5cf14ee94f51a413376aef647b.png)

this is what we currently have...

**Comment by adambreznicky** [comment]
<a href="https://github.com/ThorTheDestroyer"><img src="https://avatars1.githubusercontent.com/u/8398036?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [ThorTheDestroyer](https://github.com/ThorTheDestroyer)**
_Tuesday Jan 15, 2019 at 21:04 GMT_

----

Got it. Just to clarify, we will not know the size of the data request for this iteration, right?

Historical imagery costs are based on staff time and archive resources cost recovery calculations. 

Historical search (1 image per decade): $40.00 per site
Single year search: $18.00 per site
Scanned historical frame: $10.00 per image
Scanned + georeferenced historical frame $20.00 per image
Digital Download (0-10GB) $7.00

**Comment by adambreznicky** [comment]
<a href="https://github.com/ThorTheDestroyer"><img src="https://avatars1.githubusercontent.com/u/8398036?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [ThorTheDestroyer](https://github.com/ThorTheDestroyer)**
_Tuesday Jan 15, 2019 at 21:11 GMT_

----

Data Reproduction costs are based on a combination of staff time and computer/server cost recovery calculations. 

Staff time: $46.00 per TB
Server/computer  time: $0.15 per GB

TNRIS provided USB drive
64 GB flash drive $40.00
External 1TB hard drive: $100.00

**Comment by adambreznicky** [comment]
<a href="https://github.com/ThorTheDestroyer"><img src="https://avatars1.githubusercontent.com/u/8398036?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [ThorTheDestroyer](https://github.com/ThorTheDestroyer)**
_Tuesday Jan 15, 2019 at 21:30 GMT_

----

I'll do the "maps"one tomorrow When Im back in the office.

**Comment by adambreznicky** [comment]
<a href="https://github.com/adambreznicky"><img src="https://avatars3.githubusercontent.com/u/5650145?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [adambreznicky](https://github.com/adambreznicky)**
_Tuesday Jan 15, 2019 at 21:34 GMT_

----

perfect. thanks!

we don't need the "maps" one for the data holodeck (maps order form will be staying on tnris.org) but we could definitely use it on the tnris.org form

**Comment by adambreznicky** [comment]
<a href="https://github.com/jwhaney"><img src="https://avatars1.githubusercontent.com/u/13525011?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [jwhaney](https://github.com/jwhaney)**
_Thursday Jan 17, 2019 at 16:48 GMT_

----

resolved via commit - 75ec2517892857baf71f80785513cc845d24fbcf

possible future enhancement (issue #47) can automatically calculate a users costs based on parameters filled out in the form and will show that programmatically calculated cost at the bottom before user submission.
